ui/playback_rate_selection.js

```diff
@@ -112,15 +111,14 @@ shaka.ui.PlaybackRateSelection = class extends shaka.ui.SettingsMenu {
   }
 
   /** @private */
-  addPlaybackRates_() {
-    for (const rateStr of this.playbackRates_.keys()) {
+  updatePlaybackRates_() {
+    for (const rate of this.playbackRates_.sort()) {
```

Comment:
If this is cached, do we need to sort it every time?

Reply:
Since we only call it when the component is created, it is not necessary.